tests/general/test_message_parsing.py

```diff
@@ -373,6 +373,16 @@ def test_address_parsing():
     parsed = parse_mimepart_address_header(mimepart, "To")
     assert parsed == [["", "alice@foocorp.com"], ["", "bob@foocorp.com"]]
 
+    # Multiple addresses on one header
+    mimepart = mime.from_string("To: alice@foocorp.com, bob@foocorp.com")
+    parsed = parse_mimepart_address_header(mimepart, "To")
+    assert parsed == [["", "alice@foocorp.com"], ["", "bob@foocorp.com"]]
+
+    # Non ASCII
```

Comment:
You can keep these tests, but both of these are already mostly covered in this one above, right?
```
    mimepart = mime.from_string(
        "To: =?utf-8?Q?Foo=2C=20Corp.?= <info@foocorp.com>, "
        "=?utf-8?Q?Support?= <support@foocorp.com>"
    )
```

Reply:
I specifically wanted to have something that handles non ASCII, since `=2c=20` is just `, ` encoded , it's gonna become useful once I actively start testing Python 3. I will remove the case with two addresses in one header because it's not useful indeed.